src/ILogger/Readme.md

```diff
@@ -0,0 +1,206 @@
+# ILogger
+To learn about ILogger based logging, see [this](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging) documentation.
+
+## Console Application
+
+Following shows a sample Console Application configured to send ILogger traces to application insights.
+
+Packages installed
+```xml
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="2.1.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="2.1.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging.ApplicationInsights" Version="2.9.0-beta3" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="2.1.0" />
+  </ItemGroup>
+```
+
+```csharp
+class Program
+{
+    static void Main(string[] args)
+    {
+        // Create DI container.
+        IServiceCollection services = new ServiceCollection();
+            
+        // Add the logging pipelines to use. We are using Application Insights only here.
+        services.AddLogging(loggingBuilder =>
+        {
+			// Optional: Apply filters to configure LogLevel Trace or above is sent to ApplicationInsights for all
```

Comment:
still some issues with indentation

Reply:
![image](https://user-images.githubusercontent.com/5232798/52091323-e2002e80-2568-11e9-80e1-89fd0fb86cb4.png)
I am using this online markdown editor and things look correct there.. allow me to merge now and come back to fix any indentation if still persisting.